types/creditkey-js/creditkey-js-tests.ts

```diff
@@ -0,0 +1,19 @@
+import ck from 'creditkey-js';
+
+const client = new ck.Client('my-api-key', 'development');
+const charges = new ck.Charges(100, 10, 20, 30, 100);
+const items = [
+	new ck.CartItem('product123', 'Product 123', 123.00, 'PROD123', 1, null, null)
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/creditkey-js/creditkey-js-tests.ts;
  Line: 6;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Reply:
prettified the test file, should be fixed.